java/DEVELOPER.md

```diff
@@ -242,6 +248,26 @@ To run one class, use the following command:
 ./gradlew :client:test --tests 'TransactionTests' --rerun
 ```
 
+To run IT tests against an existing cluster and/or standalone endpoint, use:
+```bash
+./gradlew :integTest:test -Dcluster-endpoints=localhost:7000 -Dstandalone-endpoints=localhost:6379
```

Comment:
Why do we have only 1 cluster endpoint?

Reply:
It is an example. Actually, glide core can detect (and connection to) all cluster nodes having address of only one of them.